When inserted image attachments,  calls textViewDidChange:

hstdt opened the pull request.

hstdt committed:
When inserting image attachments,  calls textViewDidChange:

Deadpikle commented:
Hi @hstdt! Thanks for the pull request! :)

The code looks good; however, there is an easier way to do what you want to do: `[self sendDelegateTVChanged]` is equivalent to the 3 lines of code you adjusted. If you make that change by using the function rather than the if statement, I will be happy to merge in the request!

hstdt committed:
Replace by a method already exists.

hstdt committed:
Update RichTextEditor.m

hstdt committed:
Update RichTextEditor.m

hstdt commented:
I modified. The white space plays a trick on me.😆

hstdt committed:
shouldDisplayRichTextOptionsInMenuControllerForRichTextEditor does not work

1. shouldDisplayRichTextOptionsInMenuControllerForRichTextEditor does
not work because `[super canPerformAction:action withSender:sender]`
always return YES;
2. when textView.isEditable = NO, these action should be hidden

hstdt commented:
I'll combine these commits.
